paper/tex/methods.tex

```diff
@@ -1,163 +1,164 @@
+\subsubsection{EAZY}
+\label{sec:eazy}
 %(Rongpu Zhou)
 
-\textsc{EAZY}\footnote{\url{https://github.com/gbrammer/eazy-photoz}} \citep[Easy and Accurate Photometric Redshifts from Yale,][]{Brammer:08} is a template-based photo-$z$ code that includes several features that extend the basic $\chi^2$ fit used in many template codes.  The code can fit the observed photometry with SEDs created from a linear combination of a set of templates at each redshift, and the best-fit SED is found by simultaneously fitting one, two or all of the templates by minimizing $\chi^2$. The minimized $\chi^2(z)$ is then combined with an apparent magnitude prior to obtain the posterior redshift probability distribution.  On examination of the source code for EAZY, it appears that rather than marginalizing across all templates in the $\chi^2$ calculation, \textsc{EAZY} takes only the minimum value of $\chi^2$ at each redshift.  This improper marginalization does not lead to the correct posterior distribution, an implementation issue that will need to be addressed in the future.
-%, although some argue that this is not the mathematically correct way of calculating the posteriors.
-\textsc{EAZY} can also account for the uncertainties in the templates by adding an empirically derived template error in quadrature as a function of redshift to the flux errors.
-
-In this paper we use the all-templates mode, which fits the photometric data with a linear combination of the five basis templates. We employed the $5$ basis templates described in Section~\ref{sec:buzzard}, and set the template error to zero since these same templates were used to produce the simulated catalog photometry. %\red{The quality parameter \texttt{Qz} is...}
-The likelihoods include the application of a type-independent apparent magnitude prior estimated from the training data.
-
-\subsubsection{LePhare}\label{sec:lephare}
-%(C\'ecile Roucelle, Eric Nuss, Johann Cohen-Tanugi)
+Easy and Accurate Photometric Redshifts from Yale \citep[\eazy\footnote{\url{https://github.com/gbrammer/eazy-photoz}},][]{Brammer:08} extends the basic \chisq\ fit procedure that defines template-fitting approaches.
+The algorithm models the observed photometry with a linear combination of template SEDs at each redshift.
+The best-fit SED is found by simultaneously fitting one, two, or all of the templates via \chisq\ minimization, which is distinct from marginalizing across all templates.
+The minimized \chisq\ likelihood at each redshift is then combined with an apparent magnitude prior to obtain the redshift posterior PDF.
+We note that the utilization of the best-fit SED rather than a proper marginalization does not lead to the correct posterior distribution, an implementation issue that has now been identified and will be addressed by the developers in the future.
+\eazy\ can account for uncertainty in the template set by adding in quadrature to the flux errors an empirically derived template error as a function of redshift.
```

Comment:
We turned off the template error function, maybe we should just remove this sentence.